addons/forum-user-agent/userscript.js

```diff
@@ -0,0 +1,9 @@
+export default async function ({ addon, console }) {
+  if (!window.mySettings) return;
+  const ua = window.mySettings.markupSet.find((x) => x.className);
+  ua.openWith = window._simple_http_agent = ua.openWith.replace("version", "versions");
+  if (location.pathname === "/discuss/3/topic/add/") {
```

Comment:
#6065 isn't resolved because this still only checks for adding a new topic in BaG, from what I've tested, it doesn't work anywhere else:
![kép](https://github.com/ScratchAddons/ScratchAddons/assets/150537842/0ba2aee8-b448-4a56-83c0-9fd85bfda001)
(QaS)
```suggestion
  if (location.pathname.match(/^(?:https?:\/\/scratch\.mit\.edu\/)?(topic|\/discuss\/settings)?/))
```

Reply:
That's supposed to only replace the user-agent on posts where it's already pre-filled so it doesn't mess with post editing but clicking the button should still work.

Reply:
> That's supposed to only replace the user-agent on posts where it's already pre-filled so it doesn't mess with post editing but clicking the button should still work.

Odd, for me it didn't add the S... Gonna test again

Reply:
![kép](https://github.com/ScratchAddons/ScratchAddons/assets/150537842/6d780498-ad3a-405e-b64a-58c90a5fb8f2)
not sure what happened then